components/contract-pages/forms/mint/mint.edition.tsx

```diff
@@ -71,6 +72,13 @@ export const EditionMintForm: React.FC<IEditionMintForm> = ({ contract }) => {
       isClosable: true,
     });
   };
+
+  // TODO FIXME
+  const onSubmit = (data: any) => {
```

Comment:
no reason to have this be an extra function (with the :any) just move this as a anon function directly into the handleSubmit() of the form

Reply:
the any is still necessary otherwise we get this
![image](https://user-images.githubusercontent.com/50103937/164104371-08c1fb3b-aa86-41f9-be0b-56c038703143.png)

probably I'm missing something here